pkg/config/setup/system_probe.go

```diff
@@ -277,8 +277,8 @@ func InitSystemProbeConfig(cfg pkgconfigmodel.Setup) {
 	cfg.BindEnv(join(netNS, "max_http_stats_buffered"), "DD_SYSTEM_PROBE_NETWORK_MAX_HTTP_STATS_BUFFERED")
 	cfg.BindEnv(join(smNS, "max_http_stats_buffered"))
 	cfg.BindEnvAndSetDefault(join(smNS, "max_kafka_stats_buffered"), 100000)
-	cfg.BindEnv(join(smNS, "max_postgres_stats_buffered"))
-	cfg.BindEnvAndSetDefault(join(smNS, "max_postgres_telemetry_buffer"), 160)
+	cfg.BindEnv(join(smNS, "postgres", "max_stats_buffered"))
```

Comment:
Since we're changing this, could we add a default for it ?

Reply:
Ho, we should not do that. `applyDefault` is basically re-implementing the entire config behavior for defaults but with the wrong source. We should remove it as soon as possible. Defaults are always overwritten by values from users.

A default is different that a setting enforce at runtime by the Agent. Also. this prevent us from creating a schema for the config in the future.

Reply:
I can agree with that, but it should be done in a separate PR to eliminate usages of applyDefault